src/components/IntervalField.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useMemo, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import { useField } from 'formik'
+import { v4 } from 'uuid'
+import { Flex, Typography, SubdomainInput } from '@etvas/etvaskit'
+import { SubLabel } from './TagInput/SubLabel'
+
+export const IntervalField = ({
+  id,
+  name,
+  label,
+  placeholder,
+  error,
+  noPreserveSpace,
+  disabled,
+  separator,
+  suffix,
+  suffixSpace
+}) => {
+  const [{ value }, , { setValue }] = useField(name)
+
+  const [idLeft, idRight] = useMemo(() => {
+    const idOrNameId = id || `${name}-${v4()}`
+    return [`${idOrNameId}-left`, `${idOrNameId}-right`]
+  }, [id, name])
```

Comment:
do we need a memo here ?

Reply:
Yes, we need the useMemo hook in order to avoid generating new uuids at every render